pisa/stages/discr_sys/csv_hypersurfaces.py

```diff
@@ -0,0 +1,274 @@
+"""
+PISA pi stage to apply hypersurface fits from discrete systematics parameterizations
```

Comment:
~PISA pi~.
It would be good if some remarks could be added on why there is a separate service for reading csv files when there is already a hypersurfaces service (and/or how these differ). There is a currently fairly comprehensive, though in the case of hypersurfaces probably outdated, [readme](https://github.com/icecube/pisa/blob/master/pisa/stages/discr_sys/README.md) which one could consider for this type of documentation.

Reply:
I think this motivation (together with some statement on if and how the mathematical approach deviates from the regular hypersurface code/service) would be good to have in the code base itself, perhaps in the csv_hypersurfaces.py module docstring.